editor/src/document/properties_panel_message_handler.rs

```diff
@@ -564,8 +558,19 @@ fn node_section_fill(fill: &Fill) -> Option<LayoutRow> {
 }
 
 fn node_section_stroke(stroke: &Stroke) -> LayoutRow {
-	let color = stroke.color();
-	let weight = stroke.width();
+	// TODO: Cleanup
+	let internal_stroke1 = stroke.clone();
+	let internal_stroke2 = stroke.clone();
+	let internal_stroke3 = stroke.clone();
+	let internal_stroke4 = stroke.clone();
+	let internal_stroke5 = stroke.clone();
+	let internal_stroke6 = stroke.clone();
+	let internal_stroke7 = stroke.clone();
+	let internal_stroke8 = stroke.clone();
+	let internal_stroke9 = stroke.clone();
+	let internal_stroke10 = stroke.clone();
+	let internal_stroke11 = stroke.clone();
```

Comment:
Somewhat non ideal but I'm not sure of a better way to do this.

Reply:
Could you maybe add a short comment explainig why this has to be done?
"We have to create multiple variables because they get moved in the closure" or something like that